src/course-team/CourseTeam.jsx

```diff
@@ -175,4 +175,4 @@ CourseTeam.propTypes = {
   courseId: PropTypes.string.isRequired,
 };
 
-export default injectIntl(CourseTeam);
+export default CourseTeam;
```

Comment:
OK, I did make this one tiny fix to a non-test file. But all the other changes are only in tests.

This component was using both `useIntl` and `injectIntl` and obviously only needs `useIntl`.